Assets/Scripts/UI/Canvas/PhotoBookSystem/PhotoBook.cs

```diff
@@ -1,6 +1,7 @@
-﻿using UnityEngine;
+﻿using UI.Canvas.PhotoTaking;
+using UnityEngine;
```

Comment:
```suggestion
using UnityEngine;

﻿using UI.Canvas.PhotoTaking;
```
Zelfde als hierboven.